clickhouse/columns/string.h

```diff
@@ -27,6 +27,9 @@ class ColumnFixedString : public Column {
             Append(v);
     }
 
+    /// Increase the capacity of the column for large block insertion.
+    void Reserve(size_t) override {};
```

Comment:
Please add implementation, it is trivial:
```C++
data_->resize(string_size_ * new_cap, '\0');
```

And also we'll need an explicit counter for a current number of items in the column, rather than implicit (`data_->size() / string_size_`)